### PR TITLE
Feat(eos_designs): Add support to enable sflow on l3 interfaces

### PR DESCRIPTION
**Pull request by MaheshGSLAB**
## Change Summary

Add support to enable sflow on l3 interfaces from node_type and network services.

## Related Issue(s)

Fixes # https://github.com/aristanetworks/avd-internal/issues/162

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Add support to enable sflow on l3 interfaces from node_type and network services.

## How to test
Run molecule

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml [hunk 1]
```diff
@@ -11,6 +11,15 @@ evpn_multicast: true
 # overlay_rd_type:
 #   admin_subfield: overlay_loopback_ip
 
+# Test for sflow on l3_interfaces from network services
+sflow_settings:
+  destinations:
+    - destination: 10.10.10.12
+      vrf: sflowvrf
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml]:
```suggestion
      vrf: sflow_vrf
```

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/node-type-l3-interfaces.yml [hunk 2]
```diff
@@ -43,3 +43,12 @@ l3_interface_profiles:
           input: TEST_POLICY
     raw_eos_cli: |
       ! TEST RAW_EOS_CLI
+
+# Test for sflow on l3_interfaces from node_type
+sflow_settings:
+  destinations:
+    - destination: 10.10.10.12
+      vrf: sflowvrf
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/node-type-l3-interfaces.yml]:
```suggestion
      vrf: sflow_vrf
```

**Review by Vibhu-gslab — APPROVED** [review]

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml (hunk 1 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml]: update at both places

**Review by Shivani-gslab — APPROVED** [review]
LGTM

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**